chatgpt_wrapper/chatgpt.py

```diff
@@ -37,24 +42,28 @@ class ChatGPT:
     session_div_id = "chatgpt-wrapper-session-data"
 
     def __init__(self, headless: bool = True, browser="firefox", model="default", timeout=60, debug_log=None, proxy: Optional[ProxySettings] = None):
+        self.model = model
         self.log = self._set_logging(debug_log)
         self.log.info("ChatGPT initialized")
-        self.play = sync_playwright().start()
+        asyncio.run(self.create(headless, browser, timeout, proxy))
```

Comment:
Using a nested asyncio seems not that elegant...
By searching on stackoverflow I believe something like this might be better:
```python
class ChatGPT:
    def __init__(self,...):
        # Only do self.xxx=None
        pass
    @classmethod
    async def create(cls):
        self=cls()
        # Remaining setup...
```

Besides, by having this line (`asyncio.run(self.create(headless, browser, timeout, proxy))`) here, will it block the event loop? 

By my understanding, if you `await` a coroutine, it actively gives the CPU time to others (rather than being scheduled by the OS) before that coroutine is done. But if you call `asyncio.run(xxx)` in an **async** context, will it be able to give CPU time to others?

From [stackoverflow](https://stackoverflow.com/questions/59740704/correct-use-constraints-of-use-of-nest-asyncio):

> The monkey patch nest_async tries to break that design principle, but it doesn't do a very good job of it as **nested runs will not give time to tasks scheduled outside the nested run**. This can potentially leading to starvation if you spend too much time on code inside the nested run.

Hmm... That doesn't seem good

Reply:
> Using a nested asyncio seems not that elegant...

100% agree, but prompt_toolkit creates and event loop, and so does Playwright, and this is (so far), the only way I found to make them play together.

Perhaps it's possible to pass an existing loop into one or the other, I haven't dug that deep. Otherwise it seems like it would require a change to one or the other of those packages.

Clearly I need to read up on async programming in Python ;)

I'm open to a better fix if you have one.